src/main/java/com/denizenscript/depenizen/bukkit/commands/quests/QuestsCommand.java

```diff
@@ -131,8 +131,7 @@ public void execute(ScriptEntry scriptEntry) {
             Debug.report(scriptEntry, getName(), action.debug() 
                     + (questId != null ? questId.debug() : "") 
                     + (stageNum != null ? stageNum.debug() : "") 
-                    + (points != null ? points.debug() : "") 
-                    + (state != null ? state.debug() : ""));
+                    + (points != null ? points.debug() : ""));

```

Comment:
Er... you removed the entire debug output, instead of just the null check...

Reply:
My mistake. Thanks!